pkgs/os-specific/linux/fbterm/default.nix

```diff
@@ -9,7 +9,8 @@ let
     url="http://fbterm.googlecode.com/files/fbterm-1.7.0.tar.gz";
     sha256="0pciv5by989vzvjxsv1jsv4bdp4m8j0nfbl29jm5fwi12w4603vj";
   };
-  buildInputs = [gpm freetype fontconfig ncurses libx86];
+  buildInputs = [gpm freetype fontconfig ncurses libx86]
+    ++ lib.optional (stdenv.isi686 || stdenv.isx86_64) libx86;
```

Comment:
This is the same thing isn't it? Just adding libx86 a second time?

Reply:
That would be a cherry-pick gone wrong, whoops! Fixed.